Makefile

```diff
@@ -17,7 +17,7 @@ generate-openapi-client:
 
 .PHONY: fetch-latest-spec
 fetch-latest-spec:
-	curl -L https://management-staging.crdb.io/assets/docs/api/latest/openapi.json > internal/spec/openapi.json
+	curl -L http://localhost:7900/assets/docs/api/latest/openapi.json > internal/spec/openapi.json
```

Comment:
Could you actually change this back to https://cockroachlabs.cloud, please? I didn't mean to change it to staging, and it shouldn't be localhost either.

Reply:
Oops, didn't mean to commit that 🤦🏽‍♂️

Fixed!